ui/lib/core/addon/templates/components/field-group-show.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="box is-shadowless is-sideless is-fullwidth is-marginless">
+  {{#each @model.fieldGroups as |fieldGroup|}}
+    {{#each-in fieldGroup as |group fields|}}
+      {{#if (or (eq group "default") (eq group "Options"))}}
+        {{#each fields as |attr|}}
+          <InfoTableRow
+            @alwaysRender={{showAllFields}}
+            @label={{capitalize (or attr.options.label (humanize (dasherize attr.name)))}}
```

Comment:
I think we may no longer need to do both `humanize` and `dasherize` since openApi gives us `displayNames` now, but we could also leave this in here just in case there are new fields that don't have them. 🤔

Reply:
It's the fallback here - that's what the `or` is doing - so I think it's worth keeping. `attr.options.label` gets populated from Open API's display name struct.